R/plot_sankey.R

```diff
@@ -26,11 +21,10 @@
 #'
 #' # Plot with best_case weight
 #' plot_sankey(financial, mode = "best_case")
-plot_sankey <- function(data, with_company = TRUE, mode = c("equal_weight", "worst_case", "best_case", "main_activity")) {
+plot_sankey <- function(data, with_company = TRUE, mode = c("equal_weight", "worst_case", "best_case")) {
```

Comment:
The style here would be this:

```r
plot_sankey <- function(data,
                        with_company = TRUE,
                        mode = c("equal_weight", "worst_case", "best_case")) {
  # ...
}
```

The rest of the cases I'll leave it to you.